task/control-plane-agent/src/mgs_gimlet/host_phase2.rs

```diff
@@ -181,11 +180,16 @@ impl HostPhase2Requester {
 
         let n = gateway_messages::serialize(tx_buf, &message).unwrap_lite();
 
+        let vid = match port {
+            SpPort::One => task_net_api::VLanId::Sidecar2,
```

Comment:
What determines this, and is the 1=>2; 2=>1 here deliberate? (If so, a comment should do it.)

Reply:
Yup, this is wrong; fixed